### ISSUE TITLE
How to use it on Webpack2/stylelint-webpack-plugin ?

### ISSUE THREAD

**Issue by soykje** [AleshaOleg/postcss-sass#11]
Hi,

I'm trying for days to use Stylelint on a Webpack2 project, to lint my .sass files (not .scss). But in any ways, I'm not able to get it working... Is there something I'm missing? Here is my last try (but I can't find out how to fix it):

**webpack.config.json**
```
const path = require('path')
const webpack = require('webpack')

const ExtractTextPlugin = require('extract-text-webpack-plugin')
const StyleLintPlugin = require('stylelint-webpack-plugin')

module.exports = {
  entry: ['./src/index.js', './src/styles/index.sass'],
  output: {
    filename: 'bundle.js',
    path: path.resolve(__dirname, 'dist'),
    publicPath: '/dist/'
  },
  module: {
    rules: [
      {
        test: /\.js$/,
        exclude: /node_modules/,
        use: ['babel-loader', 'eslint-loader']
      },
      {
        test: /\.(sass|scss)$/,
        use: ExtractTextPlugin.extract({
          fallback: 'style-loader',
          use: [
            'css-loader',
            { loader: 'postcss-loader',
              options: {
                plugins: [
                  require('autoprefixer')()
                ]
              }
            },
            'sass-loader'
          ]
        })
      }
    ]
  },
  plugins: [
    new webpack.ProvidePlugin({
      $: 'jquery',
      jQuery: 'jquery'
    }),
    new ExtractTextPlugin({
      filename: 'styles.css',
      disable: false,
      allChunks: true
    }),
    new StyleLintPlugin({
      failOnError: true,
      quiet: false
    }),
    new webpack.DefinePlugin({
      'process.env': {
        NODE_ENV: JSON.stringify('production')
      }
    }),
    new webpack.optimize.UglifyJsPlugin()
  ],
  devServer: {
    contentBase: path.join(__dirname, '/')
  }
}
```

**.stylelintrc**
```
{
  "extends": "stylelint-config-standard",
  "plugins": ["postcss-sass"],
  "rules": {
    "indentation": 2
  }
}
```

**Results**
```
Error: stylelint v3+ requires plugins to expose a ruleName. The plugin "/*/*/*/*/node_modules/postcss-sass/index.js" is not doing this, so will not work with stylelint v3+. Please file an issue with the plugin.
    at module.exports (/*/*/*/*/node_modules/stylelint/lib/utils/configurationError.js:8:27)
    at /*/*/*/*/node_modules/stylelint/lib/augmentConfig.js:255:15
    at Array.forEach (native)
    at /*/*/*/*/node_modules/stylelint/lib/augmentConfig.js:253:28
    at Array.reduce (native)
    at addPluginFunctions (/*/*/*/*/node_modules/stylelint/lib/augmentConfig.js:244:45)
    at /*/*/*/*/node_modules/stylelint/lib/augmentConfig.js:76:12
error Command failed with exit code 1.
```

Thx in advance for your answer! :+1:

**Comment by AleshaOleg** [comment]
Hey, unfortunately stylelint is not have support sass syntax yes. [Here](Unfortunately) is more details:) Any help, will be appreciated. Thanks, for your question!

**Comment by AleshaOleg** [comment]
@soykje Hey! `stylelint` team released `9.0.0` version two days ago, with experimental Sass support.
You can ping guys from `stylelint-webpack-plugin` to update it too or force update stylelint manually in your local `node_modules` and try to use webpack with `.sass` files. If it still not working or you have some errors, please create a new issue and describe your problems. Thanks!
